### PR TITLE
Delete Multiple Identities 

### LINKED ISSUE #1447: As a user, I want to be able to delete multiple identities
## Description

We should let users delete multiple identities at once from the command palette.
This will make it easier to cleanup their wallets if they have lots of identities.
## Steps to Reproduce

1.
2.
3.

## Existing issues

- [ ] [Stack Overflow issues](https://stackoverflow.com/questions/tagged/ibp-vscode-extension)
- [ ] [GitHub Issues](https://github.com/IBM-Blockchain/blockchain-vscode-extension/issues)

## Your Environment

* IBM Blockchain Platform Extension Version:
* Visual Studio Code version:
* Environment name and version (e.g. Node.js v8.12.0, npm v6.4.1):
* Operating System and version:
* Link to your project:

### PR DESCRIPTION
**Pull request by toggle007**
Added functionality to allow the user to delete multiple identities at once.
Also written tests to check the additional piece of code.

Closes #1447.

Signed-off-by: Akshat Shah <akshatshah@akshats-mbp.hursley.uk.ibm.com>

### CODE REVIEW

File: extension/commands/UserInputUtil.ts [hunk 1]
```diff
@@ -256,11 +256,11 @@ export class UserInputUtil {
         return dir;
     }
 
-    public static showIdentitiesQuickPickBox(prompt: string, identities: string[], showCreate: boolean = false): Thenable<string> {
+    public static showIdentitiesQuickPickBox(prompt: string, canPickMany: boolean, identities: string[], showCreate: boolean = false): Thenable<string | string[] | undefined> {
```
Comment (cazfletch) [inline extension/commands/UserInputUtil.ts]: you don't need undefined here

File: extension/commands/deleteIdentityCommand.ts [hunk 2]
```diff
@@ -63,8 +63,8 @@ export async function deleteIdentity(treeItem: IdentityTreeItem): Promise<void>
             return;
         }
 
-        identityName = await UserInputUtil.showIdentitiesQuickPickBox('Choose the identity to delete', identityNames);
-        if (!identityName) {
+        identitiesToDelete = await UserInputUtil.showIdentitiesQuickPickBox('Choose the identities you wish to delete', true, identityNames) as string[];
```
Comment (cazfletch) [inline extension/commands/deleteIdentityCommand.ts]: Can this just be `Choose the identities to delete`?

File: extension/commands/UserInputUtil.ts (hunk 1 above)
Comment (toggle007) [reply extension/commands/UserInputUtil.ts]: Fixed

File: extension/commands/deleteIdentityCommand.ts (hunk 2 above)
Comment (toggle007) [reply extension/commands/deleteIdentityCommand.ts]: Changed it

**Pull request merged**